core/designsystem/src/main/java/com/mifos/core/designsystem/component/MifosFAB.kt

```diff
@@ -0,0 +1,27 @@
+package com.mifos.core.designsystem.component
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material3.FloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import com.mifos.core.designsystem.theme.BlueSecondary
+
+@Composable
+fun MifosFAB(
```

Comment:
designsystem package should only contains the customer component like TextField, Button etc and custom generic ui should go to `ui` module.

And Please wrap this around the box and align it to bottom right so you can directly use anywhere.

Reply:
Moved to UI Module